simde/x86/sse4.2.h

```diff
@@ -275,15 +279,19 @@ simde_mm_crc32_u8(uint32_t prevcrc, uint8_t v) {
   #if defined(SIMDE_X86_SSE4_2_NATIVE)
     return _mm_crc32_u8(prevcrc, v);
   #else
-    uint32_t crc = prevcrc;
-    crc ^= v;
-    for(int bit = 0 ; bit < 8 ; bit++) {
-      if (crc & 1)
-        crc = (crc >> 1) ^ UINT32_C(0x82f63b78);
-      else
-        crc = (crc >> 1);
-    }
-    return crc;
+    #if defined(SIMDE_ARM_NEON_A64V8_NATIVE)
```

Comment:
You also need to check `defined(__ARM_FEATURE_CRC32)` before calling the `__crc32*`, not just before including the header.  If `__ARM_FEATURE_CRC32` isn't set some compilers will not define the functions, which is why this is still giving an error on the AArch64 builds on Drone and Travis.

You should probably also add at least one AArch64 build to include CRC32 support (`-march=armv8-a+simd+crc` instead of just `-march=armv8-a+simd`).  See https://github.com/simd-everywhere/simde/blob/de8b41f876cf44a184e16ec57e4ed0d5c49f7944/.travis.yml#L75

Reply:
fixed this